neco-server-exporter/e2e/Makefile

```diff
@@ -0,0 +1,80 @@
+ARCH ?= amd64
+OS ?= linux
+
+E2ETEST_K8S_VERSION  := 1.33.4
+KIND_VERSION := 0.30.0
+CILIUM_VERSION := $(shell go list -m -f '{{.Version}}' github.com/cilium/cilium | cut -c2-)
+CILIUM_CLI_VERSION := 0.18.8
+
+PROJECT_DIR := $(CURDIR)/../
+BIN_DIR := $(PROJECT_DIR)/bin
+
+CURL := curl -sSLf
+KUBECTL := $(BIN_DIR)/kubectl
+
+KIND := $(BIN_DIR)/kind
+KIND_CLUSTER_NAME := neco-server-exporter
+KIND_CONFIG := testdata/kind.yaml
+
+CILIUM_CLI := $(BIN_DIR)/cilium
+
+IMAGE_TAG ?= ghcr.io/cybozu/neco-server-exporter:dev
+
+.PHONY: setup
+setup: $(KUBECTL) $(KIND) $(CILIUM_CLI)
+
+.PHONY: prepare
+prepare:
+	sudo sysctl -w kernel.bpf_stats_enabled=1
+
+.PHONY: wait-for-workloads
+wait-for-workloads:
+	$(KUBECTL) wait --for=condition=Available --all deployments --all-namespaces --timeout=1h
+	$(KUBECTL) wait --for=condition=Ready --all pods --all-namespaces --timeout=1h
+
+.PHONY: start
+start:
+	$(KIND) create cluster --name=$(KIND_CLUSTER_NAME) --image=kindest/node:v$(E2ETEST_K8S_VERSION) --config=$(KIND_CONFIG)
+	$(CILIUM_CLI) install --version $(CILIUM_VERSION) --wait
+
+.PHONY: stop
+stop:
+	$(KIND) delete cluster --name=$(KIND_CLUSTER_NAME)
+
+.PHONY: install
+install:
+	$(MAKE) -C ../ IMAGE_TAG=$(IMAGE_TAG) docker-build
+	$(KIND) load docker-image $(IMAGE_TAG) --name=$(KIND_CLUSTER_NAME)
+	$(KUBECTL) apply -f testdata/namespace.yaml
+	$(KUBECTL) apply -f testdata/daemonset.yaml
+	$(KUBECTL) apply -f testdata/service.yaml
+	$(MAKE) --no-print-directory wait-for-workloads
+
+.PHONY: uninstall
+uninstall:
+	$(KUBECTL) delete -f testdata/namespace.yaml
+	-docker image rm $(IMAGE_TAG)
+	-docker image prune -f
```

Comment:
Isn't it enough `docker image rm $(IMAGE_TAG)`?
`image prune` seems little dangerous to me.